editor/plugins/canvas_item_editor_plugin.h

```diff
@@ -553,14 +540,11 @@ class CanvasItemEditor : public VBoxContainer {
 	HSplitContainer *palette_split;
 	VSplitContainer *bottom_split;
 
-	bool bone_list_dirty;
-	void _queue_update_bone_list();
-	void _update_bone_list();
-	void _tree_changed(Node *);
-
 	void _popup_warning_temporarily(Control *p_control, const float p_duration);
 	void _popup_warning_depop(Control *p_control);
 
+	void _set_owner_for_node_and_children(Node *node, Node *owner);
```

Comment:
There are still lots of parameters that are missing the `_p` prefix in several headers (coding style).